autophonepulsemonitor.py

```diff
@@ -521,14 +518,25 @@ def get_treeherder_job(self, project, job_id):
             self.treeherder_url, project, job_id)
         return utils.get_remote_json(url)
 
-    def get_treeherder_privatebuild_artifact(self, job):
-        if job:
-            for artifact in job['artifacts']:
-                if artifact['name'] == 'privatebuild':
-                    url = '%s%s' % (
-                        self.treeherder_url, artifact['resource_uri'])
-                    return utils.get_remote_json(url)
-        return None
+    def get_treeherder_privatebuild_info(self, project, job):
+        url = '%s%s' % (
+            self.treeherder_url,
+            '/jobdetail/?repository=%s&job_id=%s' % (
+                project, job['id')))
```

Comment:
I don't get the benefit of this nested string interpolation. Can we do it all at once like 

        url = '%s/jobdetail/?repository=%s&job_id=%s' % (
            self.treeherder_url, project, job['id'])